.idea/anyway-frontend.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add the .idea folder to gitignore

Reply:
Done